gitmessage

```diff
@@ -1,6 +1,36 @@
+# <type>: (If applied, this commit will...) <subject> (Max 50 char)
+# |<----  Using a Maximum Of 50 Characters  ---->|
 
 
-Co-authored-by: Hugo Frappier <14030+frahugo@users.noreply.github.com>
-Co-authored-by: Nicolas Charlery <bokay@users.noreply.github.com>
-Co-authored-by: Nicholas Henry <40253+nicholasjhenry@users.noreply.github.com>
-Co-authored-by: Tetiana Dushenkivska <7498434+tetiana12345678@users.noreply.github.com>
+# Explain why this change is being made
+# |<----   Try To Limit Each Line to a Maximum Of 72 Characters   ---->|
+
+# Provide links or keys to any relevant tickets, articles or other resources
+# Example: Github issue #23
```

Comment:
Do we want to always link to the GitHub issue? Maybe only if it is an external GitHub issue.

Reply:
@frahugo Correct, we don't want to always link to the ticket". I think that is the intent here is "Provide links or keys to any relevant tickets".